src/chunk.c

```diff
@@ -1153,11 +1153,62 @@ ts_chunk_create_only_table(Hypertable *ht, Hypercube *cube, const char *schema_n
 	return chunk;
 }
 
+#if PG14_GE
+#define OSM_CHUNK_INSERT_CHECK_HOOK "osm_chunk_insert_check_hook"
+typedef int (*TS_OSM_CHUNK_INSERT_HOOK_TYPE)(Oid ht_oid, int64 range_start, int64 range_end);
+static TS_OSM_CHUNK_INSERT_HOOK_TYPE
+get_osm_chunk_insert_hook()
+{
+	TS_OSM_CHUNK_INSERT_HOOK_TYPE *func_ptr =
+		(TS_OSM_CHUNK_INSERT_HOOK_TYPE *) find_rendezvous_variable(OSM_CHUNK_INSERT_CHECK_HOOK);
+	return *func_ptr;
+}
+#endif
```

Comment:
```suggestion
#if PG14_GE
#define OSM_CHUNK_INSERT_CHECK_HOOK "osm_chunk_insert_check_hook"

/* chunk insert hook type defintion*/
typedef int (*ts_osm_chunk_insert_hook_type) (Oid ht_oid, int64 range_start, int64 range_end);

static ts_osm_chunk_insert_hook_type
get_osm_chunk_insert_hook()
{
	ts_osm_chunk_insert_hook_type *func_ptr =
		(ts_osm_chunk_insert_hook_type *) find_rendezvous_variable(OSM_CHUNK_INSERT_CHECK_HOOK);
	return *func_ptr;
}
#endif
```
IMHO is better to use a lower case for hook type names.

Reply:
fixed